src/js/index.js

```diff
@@ -47,7 +47,10 @@ const hashTitles = {
     "#saved-sets": "Saved Sets",
     "#admin": "Admin Portal",
     "#search": "Browse Sets",
-    "#account": "My Account"
+    "#account": "My Account",
+    "#discord":"Discord",
+    "#donate": "Donate",
+    "#youtube": "Youtube"
```

Comment:
Oops - I didn't catch this:
You have to add the credits page too
```suggestion
    "#youtube": "Youtube",
    "#credits": "Credits"
```

Reply:
But YouTube isn't a page.... it's on Discord